README.md

```diff
@@ -1,6 +1,8 @@
 # appimagelint
 
-appimagelint is a tool to check AppImages for common issues.
+`appimagelint` is a tool to check [AppImage](https://appimage.org/) files for common issues.
```

Comment:
I don't like the app's name monospaced, it's not like the app name and the command you call differ at all.